sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -28,6 +30,16 @@ def stream_json_error():
 def stream_compressed_header_error():
     yield b'test'
 
+def stream_decompress_header():
+    try:
+        file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "./files/test.tar.gz"))
+        with open(file_path, "rb") as fd:
+            yield fd.read()
+    except:
+        file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", "./files/test.tar.gz"))
+        with open(file_path, "rb") as fd:
+            yield fd.read()
+
```

Comment:
@zhzhh97 I think you're hitting the same error in your PR as we see in Marcus's PR (CI isn't running on this PR because it looks like there is a merge conflict). See this comment for a possible resolution: https://github.com/Azure/azure-sdk-for-python/pull/22624#discussion_r812512287